runtime/jcl/common/mgmtthread.c

```diff
@@ -1317,15 +1322,25 @@ getThreadInfo(J9VMThread *currentThread, J9VMThread *targetThread, ThreadInfo *i
 	j9object_t monitorOwnerObject = NULL;
 	IDATA exc = 0; /* exception index */
 
+#if JAVA_SPEC_VERSION >= 19
+	J9VMThread stackThread = {0};
+	J9VMEntryLocalStorage els = {0};
+	J9VMContinuation *continuation = targetThread->currentContinuation;
+	j9object_t threadObject = currentThread->carrierThreadObject;
+#else /* JAVA_SPEC_VERSION >= 19 */
+	j9object_t threadObject = currentThread->threadObject;
+#endif /* JAVA_SPEC_VERSION >= 19 */
+
 	Trc_JCL_threadmxbean_getThreadInfo_Entry(currentThread, targetThread);
 
-	info->thread =
-		vmfns->j9jni_createLocalRef((JNIEnv *)currentThread, (j9object_t)targetThread->threadObject);
+	info->thread = vmfns->j9jni_createLocalRef((JNIEnv *)currentThread, threadObject);
 	/* Set the native thread ID available through the thread library. */
 	info->nativeTID = (jlong) omrthread_get_osId(targetThread->osThread);
 	info->vmstate = getVMThreadObjectState(targetThread, &monitorObject, &monitorOwner, NULL);
-	if (targetThread->threadObject) {
-		info->jclThreadState = getJclThreadState(info->vmstate, J9VMJAVALANGTHREAD_STARTED(currentThread, targetThread->threadObject));
+	if (NULL != threadObject) {
+		info->jclThreadState = getJclThreadState(
+						info->vmstate,
+						J9VMJAVALANGTHREAD_STARTED(currentThread, threadObject));
```

Comment:
This can be done as one line, no need for spliting

Reply:
it will exceed the 100 char limit if put on the same line. so, the current formatting is good.